### ISSUE TITLE
CreateConversation.java returning 400 status

### ISSUE THREAD

**Issue by brianrisk** [helpscout/helpscout-api-java#8]
The code from CreateConversation.java (when a working api key is put in) throws:

Exception in thread "main" java.lang.RuntimeException: net.helpscout.api.exception.InvalidFormatException: The request was not formatted correctly

**Comment by pavelkokush** [comment]
+1
I think there are 2 problem:
- Example does not work
- ApiClient does not return actual error. Any rest clients like postman return more readable errors:

```
{
  "code": 400,
  "error": "Input could not be validated",
  "validationErrors": [
    {
      "property": "createdBy:type",
      "value": null,
      "message": "createdby:type is required"
    }
  ]
}
```

**Comment by jacobwallace** [comment]
@brianrisk @pavelkokush 

I don't have an ETA at the moment, but I plan to review all PRs and issues and create a new release. Thanks for the feedback!

**Comment by jacobwallace** [comment]
@brianrisk @pavelkokush 

Both issues mentioned here were fixed in the [1.5.0](https://github.com/helpscout/helpscout-api-java/releases/tag/v1.5.0) release.
